webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -601,9 +601,10 @@ export const GlideTable: React.FC<GlideTableProps> = ({
             );
         }
         return columnDefs[currentColumn.column];
-      }) as GridColumn[],
```

Comment:
the value should be `DataEditorProps['columns']`, but potentially it could be `undefined` due to `let dataPath: string | undefined = undefined;`. 
`as` changes the correct type to the wrong type. Instead of using `as`, used `flatMap` to treat type correctly.